nssp_demo/forecast_non_target_visits.R

```diff
@@ -0,0 +1,116 @@
+script_packages <- c(
+  "dplyr",
+  "tidyr",
+  "tibble",
+  "readr",
+  "stringr",
+  "fs",
+  "fable",
+  "jsonlite",
+  "argparser",
+  "arrow"
+)
+
+## load in packages without messages
+for (package in script_packages) {
```

Comment:
I prefer `purrr::walk` for compactness or `sapply` if you want to stay in base.

Reply:
done in e01bfd9